partcad-cli/src/partcad_cli/click/commands/add/part.py

```
@@ -44,13 +44,21 @@
 )
 @click.argument("path", type=str)  # help="Path to the file"
 @click.pass_obj
-def cli(ctx, desc, kind, provider, path):
+def cli(ctx, desc, kind, provider, path: str):
```

Ruff (0.8.2)

Missing return type annotation for public function `cli`

Add return type annotation: `None`

(ANN201)